### PR TITLE
Grab tools cascade

### PR DESCRIPTION
**Pull request by tim-heck**

### CODE REVIEW

File: app/templates/modules/tools.html [hunk 1]
```diff
@@ -1,61 +1,48 @@
+{# Grabs the fill lists of dictionarys containing the tools and resoruces from cascade #}
```
Comment (c-schwarze) [inline app/templates/modules/tools.html]: maybe put some newlines within this block? Its hard to glance through

File: app/templates/modules/tools.html [hunk 2]
```diff
@@ -1,61 +1,48 @@
...
-					<ul class="list-unstyled">
-						<li><a href="">Lynda</a></li>
-						<li><a href="">Print Services</a></li>
-						<li><a href="">Pyramed</a></li>
-						<li><a href="">Qualtrics</a></li>
-						<li><a href="">Symplicity</a></li>
-					</ul>
-				</div>
-			</div>
+            {% for i in range(2) %}
```
Comment (c-schwarze) [inline app/templates/modules/tools.html]: Maybe mention why this is range(2) and why the one below is range(3)

Comment (c-schwarze) [inline app/templates/modules/tools.html]: Also i and j are not the best variable names

File: app/templates/modules/tools.html [hunk 3]
```diff
@@ -1,61 +1,48 @@
...
+                <div class="col-md">
+                    <div class="px-md-3 py-md-3">
+                        <ul class="list-unstyled mb-4">
+                            {% for j in range(min(sorted_icon_length-i*icon_col_length, icon_col_length)) %}
```
Comment (c-schwarze) [inline app/templates/modules/tools.html]: lots of awesome logic, but a comment would be great!

File: app/templates/modules/tools.html [hunk 4]
```diff
@@ -1,61 +1,48 @@
...
+                        </ul>
+                    </div>
+                </div>
+            {% endfor %}
+            {% for i in range(3) %}
+            	<div class="col-md">
+                    <div class="px-md-3  py-md-3">
+                        <ul class="list-unstyled">
+                            {% for j in range(min(sorted_resources_length-i*resources_col_length, resources_col_length)) %}
+                                {% if sorted_resources_list[i*resources_col_length+j]['text'] == 'Internet Native Banner (INB)' %}
```
Comment (c-schwarze) [inline app/templates/modules/tools.html]: why are these called out? (INB, Print Services, Lynda)

File: app/views/__init__.py [hunk 5]
```diff
@@ -356,6 +356,18 @@ def logout(self):
...
+    sorted_list = []
+    if tools_or_resources == 'icons':
+        ordering_dict = {'Email': 0, 'Calendar': 1, 'Drive': 2, 'Moodle': 3, 'Banner': 4, 'Offices': 5}
+        sorted_list = sorted(session['user_icons']['bucket'], key=lambda dictionary: ordering_dict[dictionary['text']])
+    elif tools_or_resources == 'resources':
+        sorted_list = sorted(session['user_icons']['top'], key=lambda dictionary: dictionary['text'])
+    return sorted_list
+
+# Decorator for grab_tools_resources method for use in jinja
+app.jinja_env.globals.update(grab_tools_resources=grab_tools_resources)
```
Comment (c-schwarze) [inline app/views/__init__.py]: This is not actually the decorator. The decorator is the @app.whatever before the method (like the flask routes). Could you try seeing if you could get the decorator to work? Don't spend a ton of time, but at least put a todo

File: app/templates/modules/tools.html (hunk 4 above)
Comment (tim-heck) [inline app/templates/modules/tools.html]: @c-schwarze So cascade contains the full name like 'Internet Native Banner (INB)' and the hard code was just 'INB' so I added those if statements to change them. Is there a better way to do this?

Comment (c-schwarze) [inline app/templates/modules/tools.html]: Well if we are hardcoding them, we should just edit them in cascade: https://cms.bethel.edu/entity/edit.act?type=block&id=13d424c68c586513260ac29756469aff

Comment (tim-heck) [inline app/templates/modules/tools.html]: 👍

**Pull request merged**